code/game/jobs/job/_job.dm

```diff
@@ -37,6 +37,7 @@
 
 	var/announced = TRUE                  //If their arrival is announced on radio
 	var/latejoin_at_spawnpoints           //If this job should use roundstart spawnpoints for latejoin (offstation jobs etc)
+	var/forced_spawnpoint                 //If this job should spawn on specific spawnpoint only. Requires spawnpoint name as string.
```

Comment:
Would suggest changing the comment to say 'If set to a spawnpoint name, will use that spawn point for joining as this job' to be clearer.